contracts/proxies/GnosisSafeProxyFactory.sol

```diff
@@ -74,7 +74,7 @@ contract GnosisSafeProxyFactory {
         emit ProxyCreation(proxy, _singleton);
     }
 
-    /// @dev Allows to create new proxy contact, execute a message call to the new proxy and call a specified callback within one transaction
+    /// @dev Allows to create new proxy contract, execute a message call to the new proxy and call a specified callback within one transaction
```

Comment:
Maybe `Allows to create a new proxy contract` would be grammatically more correct